src/components/organisms/forms/AssignmentForm/AssignmentForm.tsx

```diff
@@ -80,11 +80,21 @@ const AssignmentForm: FC<AssignmentFormProps> = ({
   const defaultValues = useMemo(
     () => ({
       ...(deadline_at
-        ? { deadline_at: getLocalDateOjectFromUtcDateString(deadline_at) }
-        : {}),
+        ? {
+            deadline_at: getLocalDateObjectFromUtcDateString(
```

Comment:
**[blocker]** (Possibly)

Are we sure about this ? Ideally the correct deadline_at value should be set in BE and we should display what is set there. Otherwise we can end up with a situation, where we show 1 deadline, but the actual deadline is different.

I know that the business logic states that assignment deadline can't be after subproject deadline, but this is something that BE should handle, so we wouldn't end up with faulty data.

**NB** If this has already been discussed with BE and the decision was to show it like this on FE, then feel free to ignore this comment